package.json

```diff
@@ -180,14 +172,18 @@
     "husky": "^6.0.0",
     "jest": "^26.6.3",
     "jest-circus": "^26.6.3",
-    "jest-preset-stylelint": "^3.0.0",
+    "jest-preset-stylelint": "^4.0.0",
     "jest-watch-typeahead": "^0.6.3",
     "lint-staged": "^10.5.4",
     "np": "^7.5.0",
     "npm-run-all": "^4.1.5",
     "postcss-import": "^12.0.1",
-    "prettier": "^2.2.1",
+    "postcss-less": "^3.1.4",
+    "postcss-sass": "^0.4.4",
+    "postcss-scss": "^2.1.1",
+    "prettier": "2.2.1",
```

Comment:
[question] Is this `prettier` version change intentional?

```diff
-   "prettier": "^2.2.1",
+   "prettier": "2.2.1",
```

Reply:
Prettier was updating when I was updating the other dependencies, causing formatting changes in files unrelated to this pull request.

We [should be used using an exact version](https://prettier.io/docs/en/install.html).

Reply:
Thanks. I understand. 👍🏼